unit-testing-react-what-why-how/renderHook/problem/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+      '@babel/preset-env',
+      ['@babel/preset-react', {runtime: 'automatic'}],
+    ],
+  };
```

Comment:
is this what enables the local run? & if so, can I copy it as-is into other files?